### DISCUSSION TITLE
3d terrain

### DISCUSSION THREAD

**Discussion by Aidedenred6** [godotengine/godot-proposals#6955] [Editor]
Please add a function 3d terrain editor witch ships with the engine with no additional settings to do before editing as is on godot 4 we dont have a terrain editor & all the other plugins don't work would really appreciate it if we had a terrain editor that is intergated within the engine & eirks seamlessly like those of other engines like unity

**Comment by AThousandShips** [comment]
* Duplicate of #6121
